app/credentials/apikey/apikey.go

```diff
@@ -2,64 +2,37 @@ package apikey
 
 import (
 	"context"
-	"crypto/hmac"
-	"crypto/sha256"
-	"encoding/hex"
 	"fmt"
-	"net/url"
-	"path"
-	"strings"
-	"time"
 
 	"google.golang.org/grpc/credentials"
 )
 
 const (
-	IDHeader     = "tmprl-api-key-id"
-	SecretHeader = "tmprl-api-secret-key"
-
-	RequestSignatureHeader           = "tmprl-request-signature"
-	RequestSignatureAlgorithmHeader  = "tmprl-request-signature-algorithm"
-	RequestDatetimeHeader            = "tmprl-request-datetime"
-	DefaultRequestSignatureAlgorithm = "tmprl-hmac-sha256"
-
-	// requestSignatureFormat requires each of the following data on a newline
-	// KeyID
-	// RequestSignatureAlgorithm
-	// RequestDatetime
-	// ActionName
-	requestSignatureFormat = "%s\n%s\n%s\n%s"
+	AuthorizationHeader       = "authorization"
```

Comment:
Does this need to be "A" or does casing not matter

Reply:
Casing should not matter as headers are normalized in gPRC but I will make it capitalized to be consistent.